### PR TITLE
Explain all commands needed for one-off container management

### PR DESCRIPTION
**Pull request by curzolapierre**
Make the documentation about one-off clearer.

Thanks @vonox7

### CODE REVIEW

**Review by Soulou — CHANGES_REQUESTED** [review]
The right command for this is actually to use `scalingo ps` listing all containers, which command, since when they are running etc.

File: _posts/platform/app/2000-01-01-tasks.md [hunk 1]
```diff
@@ -72,13 +72,25 @@ If the `--detached` option is set, the container will be started as a background
 one-off container. In this case the container is started instantly, logs from
 the job are aggregated to the total logs of the application. 
 
+## Stop a Detached One-Off
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-tasks.md:75]: From what I remember, it stops any one-off

```suggestion
## Stop a One-Off
```

File: _posts/platform/app/2000-01-01-tasks.md [hunk 2]
```diff
@@ -72,13 +72,25 @@ If the `--detached` option is set, the container will be started as a background
...
 ```
 
+## List all One-Offs
+
+To figure out which containers (including attached and detached one-offs) are running,
+you can use the `stats` command.
+It displays then a list of all containers including the container name.
+
+```bash
+scalingo --app my-app stats
```
Comment (vonox7) [inline _posts/platform/app/2000-01-01-tasks.md:91]:
```suggestion
scalingo --app my-app ps
```

File: _posts/platform/app/2000-01-01-tasks.md [hunk 3]
```diff
@@ -72,13 +72,25 @@ If the `--detached` option is set, the container will be started as a background
...
 If it goes rogue, you can stop it with `one-off-stop` command.
 
 ```bash
 scalingo --app my-app one-off-stop one-off-1234
 ```
 
+## List all One-Offs
+
+To figure out which containers (including attached and detached one-offs) are running,
+you can use the `stats` command.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-tasks.md:87]:
```suggestion
you can use the `ps` command.
```

**Review by Soulou — APPROVED** [review]

**Pull request merged**